index.html

```diff
@@ -0,0 +1,279 @@
+    </head>
+ <body>
+ <nav class="nav">   
+      <header class="navbar navbar-expand-lg fixed-top navbarScroll">
+        
+          <div class="navbar__logo" href="#"> My Logo</div>
+          <di
+          v class="navbar__label_container">
+            <span class="__label__list_container"></span>
+            <span class="__label__list_container"></span>
+            <span class="__label__list_container"></span></div> 
+      </header>
+    </nav>
+  <main>
+      <section> 
+        <div class="navigation">  
+            <h1 class="navigation__bar">I'm Alex Asciencio. Glad to see you!</h1>
+  
+              <p class="me">
+                I'm a software developer! I can help you build a product , feature
+                or website Look through some of my work and experience! If you like
+                what you see and have a project you need coded, don't hesitate to
+                contact me.
+              </p>
+        
+          
+            <h2 class="let__connect">LET'S CONNECT</h2>
+            <div class="social-media">
+              <ul>
+                <li><a href=""><img src="img/t_icon.png" alt="Twitter icon" /></a></li>
+                <li><a href=""><img src="img/Li_icon.png" alt="LinkedIn icon" /></a></li>
+                <li><a href=""><img src="img/m_icon.png" alt="Medium icon" /></a></li>
+                <li><a href=""><img src="img/_gth_icon.png" alt="Github icon" /></a></li>
+                <li><a href=""><img src="img/finger_c.png" alt="finger icon" /></a></li>
+            
+            </ul>
+           </div>
+        </div>
+         </section>
+         <section class="section__card">
+            <div class="navigation__work">
+              <img src="img/work/port1.png" class="portfolio_f_image" alt="" />
+              <div class="portfolio__img__content">
+              <h3 class="portfolio__tonic">Tonic</h3>
+              <div class="feature__container">
+                <div class="feature__container_side">
+                  <p class="category__portfolio">CANOPY</p>  
+                </div>
+                  <div class="canopy__design"></div>
+                  <div class="canopy__layout">
+                   <p class="canopy__backend">Back End Dev</p>
+                   
+                  </div>
+                  <div class="backend__design">
+                  <p class="category__port">2015</p>
+                  </div>
+              </div>
+              <p class="navigation__paragraph">
+                A daily selection of privately personalized reads; no accounts or
+                sign-ups required</p>
+              <div>
+                <div class="skills">
+                  <div class="skills__container">
+                    <p>HTML</p>
+                  </div>
+                  <div class="skills__container">
+                    <p>CSS</p>
+                  </div>
+                  <div class="skills__container">
+                    <p>JavaScript</p><br/>
```

Comment:
Hi, @AsciencioAlex!

As suggested by the previous reviewer, I noticed that you used divs for your list of project technologies. However, the project requirements say that the list of project technologies (e.g., "JavaScript", "HTML" and "CSS") should be a list ul and each of them a li. This is important for semantic and accessibility reasons, as it helps to structure your content and make it more readable for screen readers and search engines.

To fix this issue, you should replace the divs with a ul tag and li tags and add the appropriate styles. For example:

```.html
<div> // Replace this with <ul class="skills">
  <div class="skills__container"> // Replace this with <li class="skills__container">
    <p>HTML</p>
  </div> // Replace this with </li>
  <div class="skills__container"> // Replace this with <li class="skills__container">
    <p>CSS</p>
  </div> // Replace this with </li>
  <div class="skills__container"> // Replace this with <li class="skills__container">
    <p>JavaScript</p><br/>
  </div> // Replace this with </li>
</div> // Replace this with </ul>

```
Please make this change to all project technologies lists and resubmit your project. Thank you! 😊